elastic/kibana
# Update view rule details link for the threshold rule

Closes #162890

## Summary

In this PR, I am updating the `View rule in Kibana` only for the new threshold rule, I will do the same for the rest of the rules in the following [ticket](https://github.com/elastic/actionable-observability/issues/13).

<img src="https://github.com/elastic/kibana/assets/12370520/3fdeaed1-ce79-4518-af44-47418f09b3d3" width=500 />


## 🧪 How to test

- Make sure `xpack.observability.unsafe.thresholdRule.enabled` is true in Kibana yaml
- Create a threshold rule with an email connector
- In the email, check the `View rule in Kibana`, it should open the observability rule details page
